<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelDepthColorsChooser</class>
 <widget class="QDialog" name="ModelDepthColorsChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model colors chooser</string>
  </property>
  <widget class="ColorWidget" name="color_widget" native="true">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>19</y>
     <width>180</width>
     <height>575</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="zmax_label">
      <property name="text">
       <string>Z max (m)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="zmax_spin">
      <property name="minimum">
       <double>-15000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="zmax_check">
      <property name="text">
       <string>DÃ©fault</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="zmin_label">
      <property name="text">
       <string>Z min(m)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="zmin_spin">
      <property name="minimum">
       <double>-15000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="zmin_check">
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header location="global">color_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
